package-lock.json

```diff
@@ -1,8 +1,17192 @@
 {
```

Comment:
@derz, detail: Could you revert the changes to this file? You were probably using a newer npm version than the one bundled with Node 12.